crates/sui-indexer-alt-graphql/src/api/types/snapshots/sui_indexer_alt_graphql__api__types__available_range__field_piplines_tests__registry_collect_pipelines_snapshot.snap

```diff
@@ -0,0 +1,1770 @@
+  => {}
+
+ChangeEpochTransaction.epoch
+  => {}
+
+ChangeEpochTransaction.protocolConfigs
+  => {}
+
+ChangeEpochTransaction.storageCharge
+  => {}
+
+ChangeEpochTransaction.computationCharge
+  => {}
+
+ChangeEpochTransaction.storageRebate
+  => {}
+
+ChangeEpochTransaction.nonRefundableStorageFee
+  => {}
+
+ChangeEpochTransaction.epochStartTimestamp
+  => {}
+
+ChangeEpochTransaction.systemPackages
+  => {}
+
+Checkpoint.sequenceNumber
+  => {}
+
+Checkpoint.query
+  => {}
+
+Checkpoint.artifactsDigest
+  => {}
+
+Checkpoint.digest
+  => {}
+
+Checkpoint.contentDigest
+  => {}
+
+Checkpoint.epoch
+  => {}
+
+Checkpoint.networkTotalTransactions
+  => {}
+
+Checkpoint.previousCheckpointDigest
+  => {}
+
+Checkpoint.rollingGasSummary
+  => {}
+
+Checkpoint.summaryBcs
+  => {}
+
+Checkpoint.contentBcs
+  => {}
+
+Checkpoint.timestamp
+  => {}
+
+Checkpoint.validatorSignatures
+  => {}
+
+Checkpoint.transactions
+  => {"cp_sequence_numbers", "tx_digests"}
+
+Checkpoint.transactions (filter: function)
+  => {"cp_sequence_numbers", "tx_calls", "tx_digests"}
+
+Checkpoint.transactions (filter: kind)
+  => {"cp_sequence_numbers", "tx_digests", "tx_kinds"}
+
+Checkpoint.transactions (filter: affectedAddress)
+  => {"cp_sequence_numbers", "tx_affected_addresses", "tx_digests"}
+
+Checkpoint.transactions (filter: affectedObject)
+  => {"cp_sequence_numbers", "tx_affected_objects", "tx_digests"}
+
+Checkpoint.transactions (filter: sentAddress)
+  => {"cp_sequence_numbers", "tx_affected_addresses", "tx_digests"}
+
+CoinDenyListStateCreateTransaction._
+  => {}
+
+CoinMetadata.address
+  => {}
+
+CoinMetadata.version
+  => {}
+
+CoinMetadata.digest
+  => {}
+
+CoinMetadata.balance
+  => {"consistent"}
+
+CoinMetadata.balances
+  => {"consistent"}
+
+CoinMetadata.contents
+  => {}
+
+CoinMetadata.decimals
+  => {}
+
+CoinMetadata.defaultSuinsName
+  => {"obj_versions"}
+
+CoinMetadata.description
+  => {}
+
+CoinMetadata.dynamicField
+  => {"obj_versions"}
+
+CoinMetadata.dynamicFields
+  => {"consistent"}
+
+CoinMetadata.dynamicObjectField
+  => {"obj_versions"}
+
+CoinMetadata.hasPublicTransfer
+  => {}
+
+CoinMetadata.iconUrl
+  => {}
+
+CoinMetadata.multiGetDynamicFields
+  => {"obj_versions"}
+
+CoinMetadata.multiGetDynamicObjectFields
+  => {"obj_versions"}
+
+CoinMetadata.moveObjectBcs
+  => {}
+
+CoinMetadata.multiGetBalances
+  => {"consistent"}
+
+CoinMetadata.name
+  => {}
+
+CoinMetadata.objectAt
+  => {"obj_versions"}
+
+CoinMetadata.objectBcs
+  => {}
+
+CoinMetadata.objectVersionsAfter
+  => {"obj_versions"}
+
+CoinMetadata.objectVersionsBefore
+  => {"obj_versions"}
+
+CoinMetadata.objects
+  => {"consistent"}
+
+CoinMetadata.owner
+  => {}
+
+CoinMetadata.previousTransaction
+  => {}
+
+CoinMetadata.storageRebate
+  => {}
+
+CoinMetadata.receivedTransactions
+  => {"cp_sequence_numbers", "tx_affected_addresses", "tx_digests"}
+
+CoinMetadata.regulatedState
+  => {}
+
+CoinMetadata.allowGlobalPause
+  => {}
+
+CoinMetadata.denyCap
+  => {}
```

Comment:
I suspect these are also all `consistent`! They are fetched from the `TreasuryCap` for legacy coin metadata.